scripts/sql/xavier-changes-2016_April/aatamsDB_queries_#294.sql

```diff
@@ -0,0 +1,26 @@
+﻿set search_path = aatams, public;
+
+begin;
+
+-- Change releases with positive latitudes
+UPDATE animal_release SET capture_location = ST_POINT(ST_X(capture_location), ST_Y(capture_location) * (-1))
```

Comment:
I believe most if not all of those inconsistencies are due to the fact that people put in latitude North instead of South when releasing their tags (North used to be the default in the web app).